makezip.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit -o nounset
```

Comment:
Could you  make this a variant of the normal install flow?

Reply:
I added the --modules parameter to the main install.sh script.